From a12424a913d137f377f9d5401999bdd6c7fbd8e1 Mon Sep 17 00:00:00 2001
From: Wu Xiaotian <yetist@gmail.com>
Date: Thu, 28 Feb 2019 10:46:53 +0800
Subject: [PATCH 2/2] Use CAJA_EXTENSION_DIR_SYS to save and show system caja
 extension dir

Here just show message, so a new variable used to save and show.
Remove fallback caja_extension_dir set, pkg check libcaja-extension
before, so this is not needed.
---
 configure.ac | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/configure.ac b/configure.ac
index 5f061f7..7b63336 100644
--- a/configure.ac
+++ b/configure.ac
@@ -81,20 +81,16 @@ if test -n "$with_caja_extension_dir"; then
     CAJA_EXTENSION_DIR=$with_caja_extension_dir
     custom_caja_extension_dir=true
 else
-    CAJA_EXTENSION_DIR=`$PKG_CONFIG --variable=extensiondir libcaja-extension`
-    CAJA_EXTENSION_DIR=${CAJA_EXTENSION_DIR#`$PKG_CONFIG --variable=libdir libcaja-extension`}
+    CAJA_EXTENSION_DIR_SYS=`$PKG_CONFIG --variable=extensiondir libcaja-extension`
+    CAJA_EXTENSION_DIR=${CAJA_EXTENSION_DIR_SYS#`$PKG_CONFIG --variable=libdir libcaja-extension`}
     CAJA_EXTENSION_DIR=${CAJA_EXTENSION_DIR#/}
     custom_caja_extension_dir=false
 fi
-if test -z "$CAJA_EXTENSION_DIR"; then
-    CAJA_EXTENSION_DIR='caja/extensions-2.0'
-    custom_caja_extension_dir=false
-fi
 
 if test x$custom_caja_extension_dir = xtrue; then
     AC_MSG_RESULT([${CAJA_EXTENSION_DIR}])
 else
-    AC_MSG_RESULT([${libdir}/${CAJA_EXTENSION_DIR}])
+    AC_MSG_RESULT([${CAJA_EXTENSION_DIR_SYS}])
 fi
 AM_CONDITIONAL([CUSTOM_CAJA_EXTENSION_DIR], [test x$custom_caja_extension_dir = xtrue])
 AC_SUBST(CAJA_EXTENSION_DIR)
-- 
2.21.0

